File: app/src/main/java/com/mediaplayer/listeners/BrightnessOnSeekBarChangeListener.java
package com.mediaplayer.listeners;


import android.view.WindowManager;
import android.widget.SeekBar;

import com.mediaplayer.activites.PlayFile;
import com.mediaplayer.variables.CommonArgs;

public class BrightnessOnSeekBarChangeListener implements SeekBar.OnSeekBarChangeListener {

    private PlayFile playFile = (PlayFile) CommonArgs.playFileCtx;
    private WindowManager.LayoutParams lp = playFile.getWindow().getAttributes();

    @Override
    public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
        float Progress = (float) progress / 10;
        lp.screenBrightness = Progress;
        if (Progress > 0) {
            playFile.getWindow().setAttributes(lp);
        }
    }

    @Override
    public void onStartTrackingTouch(SeekBar seekBar) {
    }

    @Override
    public void onStopTrackingTouch(SeekBar seekBar) {
        playFile.getWindow().setAttributes(lp);
    }

}

File: app/src/main/java/com/mediaplayer/listeners/BrightnessOnSeekBarChangeListener.kt
package com.mediaplayer.listeners


import android.widget.SeekBar
import com.mediaplayer.activites.PlayFile
import com.mediaplayer.variables.CommonArgs

class BrightnessOnSeekBarChangeListener : SeekBar.OnSeekBarChangeListener {

    private val playFile = CommonArgs.playFileCtx as PlayFile
    private val lp = playFile.window.attributes

    override fun onProgressChanged(seekBar: SeekBar, progress: Int, fromUser: Boolean) {
        val progress = progress.toFloat() / 10
        lp.screenBrightness = progress
        if (progress > 0) {
            playFile.window.attributes = lp
        }
    }

    override fun onStartTrackingTouch(seekBar: SeekBar) {}

    override fun onStopTrackingTouch(seekBar: SeekBar) {
        if (lp.screenBrightness < 0) {
            playFile.window.attributes = lp
        }
    }

}
